mattermost-helm/values.yaml

```diff
@@ -1,18 +1,8 @@
 global:
-  # Set siteUrl to the URL your users will use to access Mattermost
-  siteUrl: ""
-  filesAccessKey: "mattermostadmin"
-  filesSecretKey: "mattermostadmin"
   mattermostLicense: "THIS STRING SHOULD BE REPLACED WITH THE CONTENTS OF YOUR LICENSE FILE"
   features:
```

Comment:
Why move some of this stuff away from here? It seems like it's just spreading it around the config file making it harder to find instead of having it all in one place here at the top.

Reply:
- `filesAccessKey` and `filesSecretKey` was not use at all
- the `siteUrl` is only used in the config.json

global section has a lot of things it is hard to read and navigate and also most of the things there are not "global". I'm trying just to put the things in a order and in the right place.

what is your suggestion? keep the siteUrl in the global?

Reply:
Just wanting to simplify the config. Your PR spreads the usefull configuration settings all over the place. I get the scoping stuff, but wondering if it's worth it to just bring a bunch of useful globals to the top. Gitlab seems to be doing that with their chart: https://gitlab.com/charts/gitlab/blob/master/values.yaml

Reply:
right, will revert

Reply:
did the change @crspeller ptal and let me know what you think.